app/helpers/works_helper.rb

```diff
@@ -62,15 +62,15 @@ def work_parent_value(field)
     end
   end
 
-  # Passes value of series ID back to form when an error occurs on posting.
-  # Thanks to the way that series_attributes= is defined, series are saved
-  # and added to the work even before the work is saved. The only time that the
-  # series isn't added is when the work is a new record, and therefore the
-  # SerialWork can't be created.
-  def work_series_id(work)
-    if work.new_record? && (series = work.series.first)
-      series.id
-    end
+  # Determines whether or not "manage series" dropdown should appear
+  def check_series_box(work)
+    work.series.present? || (params[:work] && !(work_series_value(:id).blank? && work_series_value(:title).blank?))
+  end
+
+  # Passes value of fields for work series back to form when an error occurs on posting
+  def work_series_value(field)
+    return unless params[:work] && params[:work][:series_attributes]
+    params[:work][:series_attributes][field]
```

Comment:
I think [dig](https://api.rubyonrails.org/classes/ActionController/Parameters.html#method-i-dig) would be useful here.